src/concept-config.json

```diff
@@ -0,0 +1,27 @@
+{
+  "AMPATH": {
```

Comment:
We don't need to define organization specific like this, if we use the esm-config to define the config files, as it provides a way for each org/implementation to provide their own config.